config/routes.rb

```diff
@@ -1,5 +1,13 @@
+require 'sidekiq/web'
+
+Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+  username == 'mtom' && password == 'basic elephantitis swagger'
```

Comment:
oh, right.. well, that opens another question of how do we store and share configuration data that needs to be common across the team?